simulation/agents.py

```diff
@@ -226,10 +226,48 @@ def get_quote_insulation_elements(
 
         return insulation_quotes
 
+    def choose_n_elements_to_insulate(self):
+
+        # Derived from the VERD Project, 2012-2013. UK Data Service. SN: 7773, http://doi.org/10.5255/UKDA-SN-7773-1
+        # Based upon the choices of houses in 'Stage 3' - finalising or actively renovating
+
+        return random.choices([1, 2, 3], weights=[0.76, 0.17, 0.07])[0]
+
+    def choose_insulation_elements(
+        self, insulation_quotes: Dict[Element, float], n_elements: int
+    ) -> Dict[Element, float]:
+
+        sorted_tuples = sorted(insulation_quotes.items(), key=lambda item: item[1])
```

Comment:
Is this the same as the following:
```suggestion
        sorted_tuples = sorted(insulation_quotes.items(), key=insulation_quotes.get)
```
Happy to leave it as-is or change it to that if you think its easier to read